src/gt4sd/algorithms/core.py

```diff
@@ -438,6 +438,18 @@ def get_filepath_mappings_for_training_pipeline_arguments(
             f"{cls.__name__} artifacts not mapped for {training_pipeline_arguments}"
         )
 
+    @classmethod
+    def save_postprocess(
```

Comment:
I would rename to `def save_version_from_training_pipeline_arguments_postprocess`